README.md

```diff
@@ -1,83 +1,39 @@
-Bugsnag Middleware for Silex
-==
+# Bugsnag Notifier for Silex
 
-The Bugsnag middleware for Silex integrates into Silex PHP applications.
+[![Build Status](https://img.shields.io/travis/bugsnag/bugsnag-silex/master.svg?style=flat-square)](https://travis-ci.org/bugsnag/bugsnag-silex)
+[![StyleCI Status](https://styleci.io/repos/23802949/shield?branch=master)](https://styleci.io/repos/23802949)
 
-[Bugsnag](https://bugsnag.com) captures errors in real-time from your web,
-mobile and desktop applications, helping you to understand and resolve them
-as fast as possible. [Create a free account](https://bugsnag.com) to start
-capturing errors from your applications.
 
-The Bugsnag middleware for Silex supports Silex 1.2+ and PHP 5.3+.
+The Bugsnag Notifier for Silex gives you instant notification of errors and
+exceptions in your Silex PHP applications.
 
-Installation
---
 
-To get this middleware in to an existing project, the best way is to use
-[Composer](http://getcomposer.org).
+### Looking for 1.x?
 
-1. Add `bugsnag/bugsnag-silex` as a Composer dependency in your project's
-   [`composer.json`][composer-json] file:
+v1 of our Silex package supports PHP 5.3 and 5.4 and uses v2 of our main PHP library. You can find it on our [1.1 branch](https://github.com/bugsnag/bugsnag-silex/tree/1.1).
 
-    ```json
-    {
-      "require": {
-        "bugsnag/bugsnag-silex": "^1.1"
-      }
-    }
-    ```
 
-2. If you haven't already, download and [install Composer][composer-download]:
+## Documentation
```

Comment:
Let’s swap this section out for the Features/Getting started format like [some of the others](https://github.com/bugsnag/bugsnag-python/blob/master/README.md#features)